docs/specifications/tests/DNSSEC-TP/dnssec11.md

```diff
@@ -37,9 +37,9 @@ It is assumed that *Child Zone* is tested and reported by other test cases:
   not giving a correct DNS response for an authoritative name server, covered
   by [Basic04].
 * This test case will ignore any irregularities in fetching the DS record from
-  parent zoen, covered by [DNSSEC02].
-* This test case will ignore if the DNSKEY and SOA RRsets are signed, covered
-  by [DNSSEC08] and [DNSSEC09].
+  parent zone, covered by [DNSSEC02].
+* This test case will ignore if the DNSKEY and SOA RRsets are unsigned, covered
```

Comment:
I would keep "signed" here. DNSSEC08 and DNSSEC09 cover the tests related to DNSKEY and SOA respectively.
The phrase with "unsigned" seems odd as it seems to say that DNSSEC08 and DNSSEC09 are tests for unsigned RRsets.